app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -162,6 +164,8 @@ class BrowserFragment : Fragment(), BackHandler {
     }
 
     @Suppress("LongMethod", "ComplexMethod")
+    @UseExperimental(ObsoleteCoroutinesApi::class)
```

Comment:
Every case of this @UseExperimental(...) could be shortened to just @ObsoleteCoroutinesApi.